src/components/Calculator.js

```diff
@@ -1,21 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Calculate from '../logic/Calculate';
 import Button from './Button';
+import Display from './Display';
 
-class Calculator extends React.PureComponent {
-  render() {
-    return (
-      <div className="main">
-        <div className="display">
-          <h1 className="pixel font">0</h1>
-        </div>
-        <Button ctrls={['AC', '+/-', '%', '÷']} last={false} />
-        <Button ctrls={['7', '8', '9', 'X']} last={false} />
-        <Button ctrls={['4', '5', '6', '-']} last={false} />
-        <Button ctrls={['1', '2', '3', '+']} last={false} />
-        <Button ctrls={['0', '.', '=']} last />
-      </div>
-    );
-  }
-}
+const Calculator = () => {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158545229-7ecf249e-026d-45d0-8c3a-02f7c28aaf32.png)

Therefore:

- [ ] Please, kindly use lifecycle methods on your project, like `componentDidMout` for instance.

That way, we will meet this particular requirement.

Reply:
Gotcha 💯